cibyl/plugins/openstack/printers/colored.py

```diff
@@ -69,19 +69,23 @@ def _print_deployment_network_section(self, deployment, printer):
             printer[-1].append(deployment.network_backend)
 
         if deployment.ml2_driver.value:
-            is_empty_network = False
-            printer.add(self._palette.blue('ML2 driver: '), 2)
-            printer[-1].append(deployment.ml2_driver)
+            if deployment.ml2_driver.value != "N/A" or self.verbosity > 0:
+                is_empty_network = False
+                printer.add(self._palette.blue('ML2 driver: '), 2)
+                printer[-1].append(deployment.ml2_driver)
 
         if deployment.dvr.value:
-            is_empty_network = False
-            printer.add(self._palette.blue('DVR: '), 2)
-            printer[-1].append(deployment.dvr)
+            if deployment.dvr.value != "N/A" or self.verbosity > 0:
+                is_empty_network = False
+                printer.add(self._palette.blue('DVR: '), 2)
+                printer[-1].append(deployment.dvr)
 
         if deployment.tls_everywhere.value:
-            is_empty_network = False
-            printer.add(self._palette.blue('TLS everywhere: '), 2)
-            printer[-1].append(deployment.tls_everywhere)
+            if deployment.tls_everywhere.value != "False" or \
```

Comment:
Shouldn't we keep the consistent with the rest of the properties? TLS, like dvr or ironic features are set False if they are not used, but not if information about them could not be found

Reply:
100%